.github/CODEOWNERS

```diff
@@ -564,25 +564,25 @@
 #/<NotInRepo>/          @ms-premp @qiaozha
```

Comment:
@TravisCragg-MSFT you'll need to fix the warnings in the CI before we can merge this: https://dev.azure.com/azure-sdk/public/_build/results?buildId=4086736&view=logs&j=a460d732-23aa-5493-a411-cc406067acf8&t=2b56d21f-1cfb-5a3c-ae52-6b8b1a9ee09a&l=18

This should help:
https://github.com/Azure/azure-sdk-tools/blob/main/tools/codeowners-utils/README.md#linter-errors-and-how-to-deal-with-them